environments/contrail/install_vrouter_kmod.yaml

```diff
@@ -213,7 +213,7 @@ resources:
                 fi
               fi
             fi
-            if [[ `echo $role |awk -F"-" '{print $2}'` == "novacompute" || `hostname |awk -F"-" '{print $2}'` == "contrailtsn" ]]; then 
+            if [[ `echo $role |awk -F"-" '{print $2}'` == "compute" || `hostname |awk -F"-" '{print $2}'` == "contrailtsn" ]]; then
```

Comment:
If you use environments/contrail/roles_data.yaml for deployment the name would be still novacompute, because there is - name: HostnameFormatDefault: '%stackname%-novacompute-%index%' for Compute role.